### PR TITLE
Use let, const instead of var#1658

### LINKED ISSUE #1658: Use ES6 let, const instead of var
**Actual Behaviour**

Currently var is being used

**Expected Behaviour**

Use ES6 let, const instead of var

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1658 

Changes: Uses let and const instead of var.

Demo Link: https://pr-1658-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/__tests__/utils/ChatMessageUtils.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ it('check getCreatedMessageData util', () => {
 });
 
 it('check convertRawMessage util', () => {
-  var currentThreadID = 't_1';
-  var rawMessage = {
+  const currentThreadID = 't_1';
+  let rawMessage = {
```
Comment (akshatnitd) [inline src/__tests__/utils/ChatMessageUtils.js]: Use const

File: src/__tests__/utils/ChatMessageUtils.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import * as ChatMessageUtils from '../../utils/ChatMessageUtils';
 
 it('check getSUSIMessageData util', () => {
-  var timestamp = Date.now();
-  var message = {
+  const timestamp = Date.now();
+  let message = {
```
Comment (akshatnitd) [inline src/__tests__/utils/ChatMessageUtils.js]: Use const

File: src/components/Auth/Login/Login.react.js [hunk 3]
```diff
@@ -51,8 +51,8 @@ class Login extends Component {
   handleSubmit = e => {
     e.preventDefault();
 
-    var email = this.state.email.trim();
-    var password = this.state.password.trim();
+    let email = this.state.email.trim();
```
Comment (akshatnitd) [inline src/components/Auth/Login/Login.react.js]: Use const

**Comment by fragm3** [comment]
@akshatnitd done the changes, please review

File: src/actions/API.actions.js [hunk 4]
```diff
@@ -103,7 +103,7 @@ export function createSUSIMessage(createdMessage, currentThreadID, voice) {
   }
   let url = '';
   // Fetching local browser language
-  var locale = document.documentElement.getAttribute('lang');
+  let locale = document.documentElement.getAttribute('lang');
```
Comment (akshatnitd) [inline src/actions/API.actions.js]: This can be `const`

File: src/actions/History.actions.js [hunk 5]
```diff
@@ -41,7 +41,7 @@ export function getHistory() {
     timeout: 3000,
     async: false,
     success: function(history) {
-      var historyCognitionsCount = history.cognitions.length;
+      let historyCognitionsCount = history.cognitions.length;
```
Comment (akshatnitd) [inline src/actions/History.actions.js]: Can be `const`

File: src/components/ChatApp/MessageComposer.react.js [hunk 6]
```diff
@@ -130,7 +130,7 @@ class MessageComposer extends Component {
       stop: false,
     });
     if (this.state.result === '') {
-      var x = document.getElementById('snackbar');
+      let x = document.getElementById('snackbar');
```
Comment (akshatnitd) [inline src/components/ChatApp/MessageComposer.react.js]: Can be `const`

File: src/components/ChatApp/MessageListItem/helperFunctions.react.js [hunk 7]
```diff
@@ -267,7 +267,7 @@ export function renderTiles(tiles) {
     slidesToShow = 2;
     showArrows = false;
   }
-  var settings = {
+  const settings = {
```
Comment (akshatnitd) [inline src/components/ChatApp/MessageListItem/helperFunctions.react.js]: Can be `const`

**Comment by fragm3** [comment]
Update link:  [knowledgeable-amusement.surge.sh](knowledgeable-amusement.surge.sh)

File: src/components/ChatApp/MessageComposer.react.js (hunk 6 above)
Comment (fragm3) [reply src/components/ChatApp/MessageComposer.react.js]: Its properties are being changed

File: src/components/ChatApp/MessageListItem/helperFunctions.react.js (hunk 7 above)
Comment (fragm3) [reply src/components/ChatApp/MessageListItem/helperFunctions.react.js]: Already const

**Pull request merged**